osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -171,7 +174,7 @@ public Catcher([NotNull] Container trailsTarget, BeatmapDifficulty difficulty =
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager config)
         {
-            hitLighting = config.GetBindable<bool>(OsuSetting.HitLighting);
+            GenerateHitLighting = config.GetBindable<bool>(OsuSetting.HitLighting).Value;
```

Comment:
Does this need to change if the setting is updated? Example: If you click the checkbox on and off while watching a replay.
If so, I think it may be best to have both the config and a public setting, and do something like `bool shouldGenerateHitLighting = hitLighting.Value && GenerateHitLighting`.